### ISSUE TITLE
CEP 11 Text input cursor doesn't show in some cases on mac

### ISSUE THREAD

**Issue by kYakovenk** [Adobe-CEP/CEP-Resources#405]
Input field cursor on mac doesn't show under certain conditions. 

Here is normal function:
![Screen Recording 2021-09-27 at 3 20 25 PM](https://user-images.githubusercontent.com/39929429/134994735-9f972c2e-7416-4693-80a2-16983ff2f2ac.gif)

On docking the same window the cursor stops showing:
![Screen Recording 2021-09-27 at 3 22 01 PM](https://user-images.githubusercontent.com/39929429/134994750-881aa298-33c5-406e-8c75-0517543e3b54.gif)

This also happens on initial load of the extension if there are text input fields or if you follow down some redirects to a page that has text input fields. 
When focus switches to a different window and back, most of the time the issue is resolved.

**Comment by zhangusc** [comment]
@sujaisivanandan any update on this issue? Reproducible on the release of Illustrator prerelease and Photoshop Beta for Max

**Comment by sujaisivanandan** [comment]
@zhangusc yes this is being investigated. Will get back on this once I have more updates.

**Comment by zhangusc** [comment]
@sujaisivanandan Any update on this bug?

**Comment by apiscode** [comment]
Any update on this bug?

**Comment by ErinFinnegan** [comment]
The update is: 

> This is not part of CEP 11.4, however is being actively investigated.

**Comment by eskornd** [comment]
We also ran into this problem, the edit/password field won't have focus unless you deactivate & re-activate Illustrator (2023). 
Please consider fixing it. (screenshots attached)
<img width="567" alt="NoFocus" src="https://user-images.githubusercontent.com/50685037/210919364-0ac0705d-cdb8-4ed6-9102-78c03c6119c4.png">
<img width="567" alt="Reactivated" src="https://user-images.githubusercontent.com/50685037/210919390-ff5183cc-8260-4967-972c-a8b4021a3e9d.png">
